docker-compose.testnet.yml

```diff
@@ -0,0 +1,30 @@
+version: "3"
+
+services:
+  nbxplorer:
+    restart: always
+    ports: 
+      - 80:32838
+    expose:
+      - 32838
+    build:
+      context: .
+    environment:
+      NBXPLORER_NETWORK: ${NBXPLORER_NETWORK}
+      NBXPLORER_BIND: ${NBXPLORER_BIND}
+      NBXPLORER_NOAUTH: ${NBXPLORER_NOAUTH}
+      NBXPLORER_BTCRPCURL: ${NBXPLORER_BTCRPCURL}
+      NBXPLORER_BTCRPCUSER: ${NBXPLORER_BTCRPCUSER}
+      NBXPLORER_BTCRPCPASSWORD: ${NBXPLORER_BTCRPCPASSWORD}
+      NBXPLORER_BTCNODEENDPOINT: ${NBXPLORER_BTCNODEENDPOINT}
+      NBXPLORER_STARTHEIGHT: ${NBXPLORER_STARTHEIGHT}
+      NBXPLORER_RESCAN: ${NBXPLORER_RESCAN}
+      NBXPLORER_AWSSQSBLOCKQUEUEURL: ${NBXPLORER_AWSSQSBLOCKQUEUEURL}
+      NBXPLORER_AWSSQSTRANSACTIONQUEUEURL: ${NBXPLORER_AWSSQSTRANSACTIONQUEUEURL}
+    volumes:
+      - "nbxplorer_datadir:/datadir"
+
+volumes:
+  nbxplorer_datadir: 
+
+      
```

Comment:
I don't think this file is useful. I should probably remove the regtest one.

Reply:
I'm new to docker. Feel free to comment more